enterprise/backend/src/metabase_enterprise/audit_app/pages/common/cards.clj

```diff
@@ -59,14 +59,15 @@
 
 (def dashboards-ids
   "HoneySQL for a CTE to enumerate the dashboards for a Card. We get the actual ID's"
-  [:dash_card {:select [:card_id [(common/group-concat (h2x/cast
-                                                        (if (= (mdb.connection/db-type) :mysql) :char :text)
```

Comment:
We should probably just make this a function, this is going to blow up if we try to switch the application database type at runtime in the REPL or in tests

Reply:
Changed in https://github.com/metabase/metabase/pull/31716/commits/9c8c1a29cd2929999094c0908b225b2796e95946.